data/coc.md

```diff
@@ -0,0 +1,141 @@
+    or other events, inappropriate physical contact, and unwelcome
+    sexual attention.
+
+Violations can be reported online by sending a mail to
+[coc@in.pycon.org](mailto:coc@in.pycon.org), or to PyCon India
+volunteers at the venue.
+
+Violations of the Code of Conduct can result in expulsion from the
+conference with no refund. Please read on for a more detailed version
+of the Code of Conduct below.
+
+## The Long Form
+
+### Aim
+
+PyCon India is a community organized conference intended for
+advocating the use and adoption of the Python programming language in
+India. It is also a platform for fostering networking and
+collaboration among the Python developer community in India.
+
+We believe in the broad spectrum of individual and collective
+experience and in the inherent dignity of all people. We believe that
+amazing things happen when people from different worlds and
+world-views approach each other to create a conversation.
+
+PyCon India 2023 welcomes everyone and is dedicated to providing a
+harassment-free conference experience for everyone, regardless of age,
+gender, sexual orientation, physical appearance, disability,
+neurotype, race, religion, political opinion, culture or
+employment. We don’t tolerate harassment of attendees in any form.
+
+### Where
+
+This Code of Conduct applies to the venue of PyCon India 2023
+Conference, pre-event & post-event informal gatherings, related online
+forums and all conference related events organized by PyCon India.
+
+### Who
+
+This is applicable to speakers, participants, sponsors, volunteers and
+organizers.
+
+### Be The Best You Can Be
+
+- Be kind, sensitive, understanding and respectful to other
+    attendees.
+
+- Listen with empathy when someone has a different perspective.
+  
+- Talk to the volunteers if you notice that something could be
+    better.
+  
+- All communication should be appropriate for a professional
+    audience, including people from many different backgrounds.
+
+### Please Refrain From
+
+- Offensive verbal comments related to gender, sexual orientation,
+    disability, physical appearance, body size, race, religion
+
+- Violent threats or language directed against another person.
+
+- Swearing, insults or discriminatory/exclusionary/offensive jokes
+    or language.
+
+- Deliberate intimidation, stalking, following, harassing
+    photography or recording
+  
+- Sustained disruption of talks or other events
+  
+- Using sexually explicit or violent material in
+    presentations. Sexual language or imagery is inappropriate for all
+    aspects of the conference, including talks and exhibits.
+
+- Unwelcome sexual attention or inappropriate physical contact.
+  
+- Advocating for, or encouraging, any of the above behavior.
+  
+- Repeated harassment of others. In general, if someone asks you to
+    stop, then stop.
+  
+- Avoid use of sexualized clothing/uniforms/costumes, or otherwise
```

Comment:
I don't really like this point. This gets very judgemental very easily. 
Also, this gives someone the power to tell me what I can and cannot wear. 

But I can see why we might need it. Not sure what to do here.